solc/CommandLineParser.cpp

```diff
@@ -961,7 +963,7 @@ General Information)").c_str(),
 		if (countEnabledOptions(nonAssemblyModeOptions) >= 1)
 		{
 			auto optionEnabled = [&](string const& name){ return m_args.count(name) > 0; };
-			auto enabledOptions = boost::copy_range<vector<string>>(nonAssemblyModeOptions | boost::adaptors::filtered(optionEnabled));
+			auto enabledOptions = nonAssemblyModeOptions | ranges::cpp20::views::filter(optionEnabled) | ranges::to_vector;
```

Comment:
Is the cpp20 prefix needed here?

Reply:
no, it is not needed. I saw [https://github.com/ethereum/solidity/issues/10738#issuecomment-758266471](this comment) and I thought may be you care about it.

Reply:
I removed `cpp20 `

Reply:
Well, AFAIK `cpp20` is what ended up in C++20 so sticking to it would make a transition easier in cases where the library evolved further.

But yeah, we don't do that consistently and the full namespaced names are already too long so I'd drop it and only worry about that when we upgrade to C++20 :)